manifests/m/Microsoft/Sysinternals/Desktops/2.01/Microsoft.Sysinternals.Desktops.installer.yaml

```diff
@@ -3,13 +3,16 @@
 
 PackageIdentifier: Microsoft.Sysinternals.Desktops
 PackageVersion: 2.01
+InstallerType: portable
+Commands:
+- desktops

```

Comment:
```suggestion
- desktops /accepteula
```

Reply:
This is necessary to clear the first-time EULA window. 
![image](https://user-images.githubusercontent.com/114952565/217399106-0f28f064-9457-4b31-b6fa-8ade21305900.png)

Reply:
> 

@stephengillie I think this command doesn't work. I am checking the documentation but cannot find it.

Reply:
`Commands` isn't the right field for this, as it will cause issues on the CLI renaming the file

Reply:
Hi @adripo,
> @stephengillie I think this command doesn't work. I am checking the documentation but cannot find it.
It's displayed in the top line of the EULA window above:
![image](https://user-images.githubusercontent.com/114952565/221929105-ee0a10fc-6762-46ec-b5a2-db0e4145ec30.png)

Reply:
@Trenly
> `Commands` isn't the right field for this, as it will cause issues on the CLI renaming the file

Correct - it causes a symlink error:
```
2023-02-28 09:07:57.274 [CLI ] Caught std::exception: create_symlink: The system cannot find the path specified.: "C:\Users\user\AppData\Local\Microsoft\WinGet\Packages\Microsoft.Sysinternals.Desktops__DefaultSource\Desktops.exe", "C:\Users\user\AppData\Local\Microsoft\WinGet\Links\desktops /accepteula.exe"
```

I seem to remember using an installer switch to do this in the past, but can't get that to work in a VM this time either. I'm unsure of the appropriate way to apply this application switch.

Reply:
I tried to find a solution for this behavior, but no luck for now.